Packaging/Packaging.proj

```diff
@@ -0,0 +1,68 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="CreateStepPackage" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    
+    <!-- Change this when the runner goes up in version-->
+    <BootstrapperDirName>SonarQube.MSBuild.Runner-0.9</BootstrapperDirName>
+    
+    <!-- Needed to understand the output directory of referenced projects-->
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+
+    <TaskOutputDir>$(MSBuildThisFileFullPath)\..\..\VsoBuildSteps\$(Configuration)\</TaskOutputDir>
+    <TaskSourceDir>$(MSBuildThisFileFullPath)\..\..\VsoBuildStepSources\</TaskSourceDir>
+    <PreBuildTaskDestinationDir>$(TaskOutputDir)\SonarQubeInstallPreBuild\</PreBuildTaskDestinationDir>
```

Comment:
So the target directory for the copy is under the source folder of the copy? This seems a bit odd. Why not create a separate "Deployment" directory higher up the tree, and then have separate sub-folders for the different deployment artefacts (one for the Build vNext, one for the bootstrapper etc)?